# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fbgrab/destdir.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- fbgrab-1.0/Makefile.vanilla	2007-06-17 13:38:58.181107117 +0200
+++ fbgrab-1.0/Makefile	2007-06-17 13:39:32.427058683 +0200
@@ -3,13 +3,14 @@
 ### modular. So this is a simple gnu Makefile...
 ###
 
+prefix=/usr
+
 fbgrab: fbgrab.c
 	gcc -g -Wall fbgrab.c -lpng -lz -o fbgrab
 
 install:
-	strip fbgrab
-	install fbgrab /usr/bin/fbgrab
-	install fbgrab.1.man /usr/man/man1/fbgrab.1
+	install fbgrab $(DESTDIR)$(prefix)/bin/fbgrab
+	install fbgrab.1.man $(DESTDIR)$(prefix)/man/man1/fbgrab.1
 
 clean:
-	rm -f fbgrab *~ \#*\#
\ No newline at end of file
+	rm -f fbgrab *~ \#*\#
